jcl/src/java.base/share/classes/module-info.java.extra

```diff
@@ -36,6 +36,7 @@ exports com.ibm.gpu.spi to openj9.gpu;
 /*[IF CRIU_SUPPORT]*/
 exports openj9.internal.criu to openj9.criu;
 exports jdk.internal.misc to openj9.criu;
+exports com.ibm.security.criu;
```

Comment:
can this be a qualified exports?

Reply:
sorry, what do you mean by qualified exports?

Reply:
A quick reference - [The exports and exports… to directives and the qualified export](https://developer.ibm.com/tutorials/java-modularity-3/)

Reply:
thank you, yes a qualified exports to `openj9.criu` makes sense